index.js

```diff
@@ -2,8 +2,16 @@ const express = require('express');
 const app = express();
 const PORT = 4000;
 
+const positions = require('./controllers/positions.js')
```

Comment:
The imports (requires function) should be placed at the top of the file, it is a convention.